diff --git a/dist/src/index.js b/dist/src/index.js
index bce8fc5918215d281fb17c02fb3e721536a59d0d..3cc45c3ac7fb96aa9473a1bc9e2196618286b6ee 100644
--- a/dist/src/index.js
+++ b/dist/src/index.js
@@ -1,6 +1,6 @@
 'use strict';
 
-var web3_js = require('@solana/web3.js');
+var web3_js = require('@solana/kit');
 
 // src/generated/accounts/mint.ts
 function getMintEncoder() {
diff --git a/dist/src/index.mjs b/dist/src/index.mjs
index 716ea6ee68a31e6eb68c7a7d08548a768c272b4b..a531e20852f7e544a5d54460311a2feab5bdeed8 100644
--- a/dist/src/index.mjs
+++ b/dist/src/index.mjs
@@ -1,4 +1,4 @@
-import { getStructEncoder, getOptionEncoder, getAddressEncoder, getU32Encoder, getU64Encoder, getU8Encoder, getBooleanEncoder, getStructDecoder, getOptionDecoder, getAddressDecoder, getU32Decoder, getU64Decoder, getU8Decoder, getBooleanDecoder, combineCodec, decodeAccount, assertAccountExists, fetchEncodedAccount, assertAccountsExist, fetchEncodedAccounts, getArrayEncoder, getArrayDecoder, getEnumEncoder, getEnumDecoder, containsBytes, isProgramError, transformEncoder, AccountRole, getProgramDerivedAddress, none, getUtf8Encoder, getUtf8Decoder, upgradeRoleToSigner, isTransactionSigner as isTransactionSigner$1 } from '@solana/web3.js';
+import { getStructEncoder, getOptionEncoder, getAddressEncoder, getU32Encoder, getU64Encoder, getU8Encoder, getBooleanEncoder, getStructDecoder, getOptionDecoder, getAddressDecoder, getU32Decoder, getU64Decoder, getU8Decoder, getBooleanDecoder, combineCodec, decodeAccount, assertAccountExists, fetchEncodedAccount, assertAccountsExist, fetchEncodedAccounts, getArrayEncoder, getArrayDecoder, getEnumEncoder, getEnumDecoder, containsBytes, isProgramError, transformEncoder, AccountRole, getProgramDerivedAddress, none, getUtf8Encoder, getUtf8Decoder, upgradeRoleToSigner, isTransactionSigner as isTransactionSigner$1 } from '@solana/kit';
 
 // src/generated/accounts/mint.ts
 function getMintEncoder() {